config/application.rb

```diff
@@ -5,36 +5,38 @@
 require 'rails'
 # Pick the frameworks you want:
 require 'active_model/railtie'
-# require 'active_job/railtie'
+# require "active_job/railtie"
 require 'active_record/railtie'
+# require "active_storage/engine"
 require 'action_controller/railtie'
 require 'action_mailer/railtie'
+# require "action_mailbox/engine"
+# require "action_text/engine"
 # require "action_view/railtie"
+# require "action_cable/engine"
 # require "sprockets/railtie"
 require_relative '../lib/http_method_not_allowed'
 require_relative '../lib/statsd_middleware'
+require 'rails/test_unit/railtie'
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
 module VetsAPI
   class Application < Rails::Application
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
-
-    # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
-    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.2
```

Comment:
I had to ask/lookup the info about this: https://prathamesh.tech/2019/08/22/new-framework-defaults-in-rails/